src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TransmissionStorage.cs

```diff
@@ -196,8 +196,9 @@ private static Transmission LoadFromTransmissionFile(IPlatformFile file, out lon
             Transmission transmission = null;
             if (file.Exists)
             {
-                if (file.DateCreated > DateTimeOffset.Now.AddDays(-2)) // The injestion service rejects anything older than 2 days.
+                if (file.DateCreated > DateTimeOffset.Now.AddDays(-2)) 
                 {
+                    // The injestion service rejects anything older than 2 days.
```

Comment:
This comment needs to be with or above the if statement rather than inside it since now it looks like it applies to a different line of code.